src/deluge/gui/views/instrument_clip_view.cpp

```diff
@@ -2839,7 +2839,8 @@ void InstrumentClipView::setRowProbability(int32_t offset) {
 	uint8_t probabilityValue = noteRow->probabilityValue;
 	bool prevBase = false;
 	// Covers the probabilities and iterations
-	probabilityValue = std::clamp<int32_t>((int32_t)probabilityValue + offset, (int32_t)0, kNumProbabilityValues + 35);
+	probabilityValue = std::clamp<int32_t>((int32_t)probabilityValue + offset, (int32_t)1,
```

Comment:
This needs to start from 0 to accomodate fill

Reply:
Fixed in https://github.com/SynthstromAudible/DelugeFirmware/pull/399/commits/22de11a907d187964e282964d95c2bb7e12dc217